class.jetpack.php

```diff
@@ -7364,6 +7364,12 @@ public static function load_jetpack_gutenberg() {
 			$version
 		);
 
+		wp_localize_script(
+			'jetpack-blocks-editor',
+			'JETPACK_BLOCK_ASSETS_BASE_URL',
```

Comment:
Nit: How common is it to use this naming convention for JS variables in Jetpack? Do we want to be consistent with what we currently do in Jetpack? I've seen both camel case and snake case, but not constant case(uppercase with underscore separator).

Reply:
It's definitely not a common naming convention. I would personally lean more towards some lowercase here, but that's a personal preference based on what we usually use here in the repo.

Reply:
Updated to `Jetpack_Block_Assets_Base_Url` in 1cf03102fc9a0dcdc9ed6c4fe92f89d859545d00